src/main/kotlin/no/nav/syfo/dialogmote/database/MoteQuery.kt

```diff
@@ -31,8 +31,9 @@ const val queryGetDialogmoteListForPersonIdent =
     """
         SELECT *
         FROM MOTE
-        left join MOTEDELTAKER_ARBEIDSTAKER on MOTEDELTAKER_ARBEIDSTAKER.mote_id = MOTE.id
```

Comment:
Siden det filtreres på MOTEDELTAKER_ARBEIDSTAKER.personident er det bedre å bruke inner join enn left join.

Reply:
Sikkert fint det, men blir ikke både resultat og ytelse likt ettersom det er en 1-til-1-relasjon mellom MOTEDELTAKER_ARBEIDSTAKER  og MOTE?

Reply:
Riktig det, gjør ikke så mye fra eller til - men jeg tenker at inner join bør være "default" og så bruker man andre varianter der det er et behov for det.